### ISSUE TITLE
Error with the 1.19 version

### ISSUE THREAD

**Issue by adpeyre** [symfony/flex#928]
Hi,

It seems, there is a problem with the latest 1.x version. 
When I try to install the 1.19.0 version, an error occured with the plugin installation. 
I use composer v1.

The 1.18.7 version works.

````
You are using Composer 1 which is deprecated. You should upgrade to Composer 2, see https://blog.packagist.com/deprecating-composer-1-support/
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Package operations: 163 installs, 0 updates, 0 removals
  - Installing ocramius/package-versions (1.4.2): Loading from cache
  - Installing symfony/flex (v1.19.0): Loading from cache
Plugin installation failed (Undefined property: Composer\Installer::$updateAllowList), rolling back
  - Removing symfony/flex (v1.19.0)


  [ErrorException]
  Undefined property: Composer\Installer::$updateAllowList

````

Ty

**Comment by nicolas-grekas** [comment]
Which exact version of composer do you use?

**Comment by adpeyre** [comment]
````
composer -V
Composer version 1.10.25 2022-01-21 10:02:15
````

**Comment by nicolas-grekas** [comment]
Got it, fixed in 2744418
